### PR TITLE
Feature/elastic zap

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/components/Announcement/Popups/TransactionPopup.tsx [hunk 1]
```diff
@@ -85,6 +85,19 @@ const summaryLiquidity = (txs: TransactionDetails) => {
...
+  const extraInfo = txs.extraInfo || {}
+  const {
+    tokenAmountIn = '',
+    tokenAmountOut = '',
+    tokenSymbolIn,
+    tokenSymbolOut,
+    zapAmountIn,
+    zapSymbolIn,
+  } = extraInfo as TransactionExtraInfo2Token
+  return `You have zapped ${zapAmountIn} ${zapSymbolIn} into ${tokenAmountIn} ${tokenSymbolIn} and ${tokenAmountOut} ${tokenSymbolOut} of liquidity to the pool`
```
Comment (nguyenhoaidanh) [inline src/components/Announcement/Popups/TransactionPopup.tsx:98]: to the pool.

Comment (namgold) [reply src/components/Announcement/Popups/TransactionPopup.tsx:98]:
```suggestion
  return t`You have zapped ${zapAmountIn} ${zapSymbolIn} into ${tokenAmountIn} ${tokenSymbolIn} and ${tokenAmountOut} ${tokenSymbolOut} of liquidity to the pool.`
```

File: .swc/plugins/v7_macos_x86_64_0.101.1/374147cf3ed8b5c7a9f9c5618159f9b6ced0015c95634faece220d4a054ae7fa
Comment (namgold) [inline .swc/plugins/v7_macos_x86_64_0.101.1/374147cf3ed8b5c7a9f9c5618159f9b6ced0015c95634faece220d4a054ae7fa:1]: delete-this

File: src/hooks/elasticZap/index.ts [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+        index === 0 || !params
+          ? null
+          : aggregatorOutputs?.filter(item => JSBI.greaterThan(params.amountIn.quotient, JSBI.BigInt(item.amountIn)))?.[
+              index - 1
+            ],
+    }
+  }, [data, loadingAggregator, aggregatorOutputs, params])
+
+  // TODO: Remove
+  console.log(bestRes)
```
Comment (namgold) [inline src/hooks/elasticZap/index.ts:160]: todo

File: src/components/FeeSelector/hook.ts [hunk 3]
```diff
@@ -55,6 +55,7 @@ export const useFeeTierDistribution = (
         )
 
         const totalPositions = feeArray.reduce((total, cur) => total + cur.activePositions, 0)
+        console.log(totalPositions)
```
Comment (namgold) [inline src/components/FeeSelector/hook.ts:58]: rm

**Review by namgold — COMMENTED** [review]
lgtm

**Review by namgold — APPROVED** [review]

**Pull request merged**